.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -582,26 +585,88 @@ popd
 # INSTALL & START GRADING SCHEDULER DAEMON
 
 
-# stop the scheduler (if it's running)
+
+#############################################################
+# stop the shipper daemon (if it's running)
+systemctl is-active --quiet submitty_autograding_shipper
+is_shipper_active_before=$?
+if [[ "$is_shipper_active_before" == "0" ]]; then
+    systemctl stop submitty_autograding_shipper
+    echo -e "Stopped Submitty Grading Shipper Daemon\n"
+fi
+systemctl is-active --quiet submitty_autograding_shipper
+is_active_tmp=$?
+if [[ "$is_active_tmp" == "0" ]]; then
+    echo -e "ERROR: did not successfully stop submitty grading shipper daemon\n"
+    exit 1
+fi
+
+
+#############################################################
+# stop the worker daemon (if it's running)
+systemctl is-active --quiet submitty_autograding_worker
+is_worker_active_before=$?
+if [[ "$is_worker_active_before" == "0" ]]; then
+    systemctl stop submitty_autograding_worker
+    echo -e "Stopped Submitty Grading Worker Daemon\n"
+fi
+systemctl is-active --quiet submitty_autograding_worker
+is_active_tmp=$?
+if [[ "$is_active_tmp" == "0" ]]; then
+    echo -e "ERROR: did not successfully stop submitty grading worker daemon\n"
+    exit 1
+fi
+
+
+
+#############################################################
+# NOTE: This section is to cleanup the old scheduler -- and this code should eventually be removed
+# BEGIN TO BE DELETED
+# stop the old scheduler (if it's running)
 systemctl is-active --quiet submitty_grading_scheduler
-is_active_before=$?
-if [[ "$is_active_before" == "0" ]]; then
+is_scheduler_active=$?
+if [[ "$is_scheduler_active" == "0" ]]; then
     systemctl stop submitty_grading_scheduler
-    echo -e "Stopped Submitty Grading Scheduler Daemon\n"
+    echo -e "WARNING: Stopped Deprecated Submitty Grading Scheduler Daemon\n"
+    # launch the shipper & worker daemons on relaunch
+    is_shipper_active_before=0
+    is_worker_active_before=0
+    echo -e "WARNING: Will launch replacement Submitty Autograding Shipper & Worker Daemon\n"
 fi
-
 systemctl is-active --quiet submitty_grading_scheduler
 is_active_tmp=$?
 if [[ "$is_active_tmp" == "0" ]]; then
-    echo -e "ERROR: did not successfully stop submitty grading scheduler daemon\n"
+    echo -e "ERROR: did not successfully stop deprecated submitty grading scheduler daemon\n"
     exit 1
 fi
+# END TO BE DELETED
```

Comment:
I'd like to discuss how we want to handle these sorts of backwards incompatible changes as we go forward as this seems super brittle, especially if we were to gain more users. When should we remove this? Once we're certain everyone has upgraded? How do we know that has happened?

We should probably discuss this and come to a consensus on perhaps a "better path" for these sort of bc-breaking changes, even though I don't think this will apply to this particular MR for now.